src/documents/ComponentGrades.md

```diff
@@ -14,121 +14,87 @@ The Infusion Framework already contains several predefined component grades that
             </td>
         </tr>
         <tr>
-            <td>fluid.modelComponent</td>
-            <td>fluid.modelRelayComponent</td>
+            <td><code>fluid.viewComponent</code></td>
             <td>
-                A "model" component is already a little component that additionally provides supports for a component's model, defined in the components options, and operations on it (<a href="tutorial-gettingStartedWithInfusion/ModelComponents.md">Tutorial - Model Components</a>).
+                A "view" component is a fluid.modelComponent is bound to a DOM container node, holds a <a href="DOMBinder.md">DOM Binder</a> and supports a view (<a href="tutorial-gettingStartedWithInfusion/ViewComponents.md">Tutorial - View Components).
             </td>
         </tr>
         <tr>
-            <td>fluid.eventedComponent</td>
-            <td></td>
+            <td><code>fluid.rendererComponent</code></td>
             <td>
-                An "evented" component is already a little component that additionally instantiates event firers based on default framework events (onCreate, onDestroy, onDetach) and events declared in the options (<a href="tutorial-gettingStartedWithInfusion/EventedComponents.md">Tutorial - Evented Components</a>).
-            </td>
-        </tr>
-        <tr>
-            <td>fluid.standardComponent</td>
-            <td>fluid.standardRelayComponent</td>
-            <td>
-                A compound of fluid.modelComponent and fluid.eventedComponent
-            </td>
-        </tr>
-        <tr>
-            <td>fluid.viewComponent</td>
-            <td>fluid.viewRelayComponent</td>
-            <td>
-                A "view" component is a fluid.standardComponent is bound to a DOM container node, holds a <a href="DOMBinder.md">DOM Binder</a> and supports a view (<a href="tutorial-gettingStartedWithInfusion/ViewComponents.md">Tutorial - View Components).
-            </td>
-        </tr>
-        <tr>
-            <td>fluid.rendererComponent</td>
-            <td>fluid.rendererRelayComponent</td>
-            <td>
-                A "renderer" component is already a vew component that bears a renderer. There are additional features provided by this component grade specified on the <a href="tutorial-gettingStartedWithInfusion/RendererComponents.md#useful-functions-and-events">Useful functions and events</a> section of the <a href="tutorial-gettingStartedWithInfusion/RendererComponents.md">Tutorial - Renderer Components</a> page
+                A "renderer" component is a view component that also bears a renderer. There are additional features provided by this component grade specified on the <a href="RendererComponents.md#useful-functions-and-events">Useful functions and events</a> section of the <a href="tutorial-gettingStartedWithInfusion/RendererComponents.md">Tutorial - Renderer Components</a> page
```

Comment:
Does it "bear" a Renderer, or does it implement some kind of lifecycle for rendering? A renderer, currently, is somewhat different from, say, a ChangeApplier, isn't it?